java/ql/lib/semmle/code/java/security/performance/RegExpTreeView.qll

```diff
@@ -0,0 +1,57 @@
+/**
+ * This module should provide a class hierarchy corresponding to a parse tree of regular expressions.
+ */
+
+import java
```

Comment:
Make the imports private? Presumably since this implements the shared lib interface we shouldn't expose anything language-specific.